docs/students/guides/google-classroom.md

```diff
@@ -0,0 +1,33 @@
+---
+id: students-google-classroom
+title: Google Classroom
+---
+
+### Joining Google Classroom
+
+When the Classroom is set up, the mentors will make the classroom's code available to everyone. You may be asked to provide your email address so you can be invited to join the class. When you sign in to your Google account and head to <a href="classroom.google.com">Google Classroom</a>, you should be able to see the class you've been invited to and you can simply click 'accept' to join. If you cannot see your class, please click the plus button and, when asked to do so, enter the class code. This will let the mentors with access to Classroom know you're trying to join and you will be given access.
```

Comment:
Can we change the link to a markdown style?

["https://classroom.google.com"](Google Classroom)

Reply:
You'll also need to add `https://` to the start :)

Reply:
Thanks for flagging this. My Markdown skills are still blossoming haha